packages/api/src/interfaces/cli/PaymentsFlat.cli.ts

```diff
@@ -8,8 +8,13 @@ import CliController from "../../shared/CliController";
 export default class PaymentsFlatCli extends CliController {
     static cmdName = "payments-flat";
 
-    async resync() {
-        this.logger.logIC("Resyncing payments flat collection");
+    async resyncExercice(exerciceBudgetaire: number) {
+        this.logger.logIC(`Resync payment flat collection for exercice ${exerciceBudgetaire}`);
+        await paymentsFlatService.updatePaymentsFlatCollection(exerciceBudgetaire);
+    }
```

Comment:
```suggestion
    resyncExercice(exerciceBudgetaire: number) {
        this.logger.logIC(`Resync payment flat collection for exercice ${exerciceBudgetaire}`);
        return paymentsFlatService.updatePaymentsFlatCollection(exerciceBudgetaire);
    }
```

à la suite du commentaire de Maxime : ça se présente comme ça. La clé pour que ça pète pas c'est de retourner la promesse, et comme ça elle est attendue par la fonction qui appelle celle-ci. Ça revient au même, juste c'est un peu moins verbeux comme propose Maxime